benchmarks/benchmark_density_GaussianDensity.py

```diff
@@ -27,7 +27,7 @@ def bench_run(self, N):
 def run():
     Ns = [1000, 10000]
     width = 100
-    r_max = 10
+    r_max = 1
```

Comment:
@bdice I don't remember why these were changed, do you?

Reply:
There are two ratios, `r_max/L` and `sigma/r_max` that were somewhat unrealistic for practical use, AND the benchmark was taking too long.

Previously, `sigma=0.1` and `r_max=10` meant that `sigma/r_max` was very small, and the contributions near `r=r_max` were at the 100-sigma limit of a normal distribution (effectively zero). Changing that ratio to `sigma/r_max=10` is much more realistic. In practice it's fine to use an even lower value like 3, just depends on how precisely the integral over all grid spaces needs to get to 1. The tests in this PR explicitly check this normalization, which is related to issue #611.

The other relevant ratio is `r_max/L`, which was previously `3.1`. This meant that every point's Gaussian is contributing to (and being computed for) an extremely large proportion of grid cells. Like `4/3 * pi * (1/3.1)**3 = 0.1406`. That's not really how I've seen GaussianDensity used. I assumed a system of 10x10x10 particles in a simple cubic lattice, where every particle has a diameter of 1 (radius 0.5, diameter is equivalent to 10 bin widths since `width=100`).